apps/api/src/app/bridge/usecases/get-bridge-status/get-bridge-status.usecase.ts

```diff
@@ -18,6 +20,11 @@ export class GetBridgeStatus {
 
       return response.data;
     } catch (err: any) {
+      Logger.error(
+        `Failed to verify Bridge endpoint: ${(err as Error).message || err}`,
+        (err as Error).stack,
+        LOG_CONTEXT
+      );
```

Comment:
I suggest we include the Bridge URL in the error message to assist in isolating & debugging error messages.

```suggestion
      Logger.error(
        `Failed to verify Bridge endpoint ${command.bridgeUrl} with error: ${(err as Error).message || err}`,
        (err as Error).stack,
        LOG_CONTEXT
      );
```